sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/Aggregation.java

```diff
@@ -0,0 +1,143 @@
+          i -> {
+            switch (i.getType()) {
+              case COUNTER:
+              case UP_DOWN_COUNTER:
+              case OBSERVABLE_SUM:
+              case OBSERVABLE_UP_DOWN_SUM:
+                return AggregatorFactory.sum(AggregationTemporality.CUMULATIVE);
+              case HISTOGRAM:
+                return AggregatorFactory.histogram(
+                    DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES, AggregationTemporality.CUMULATIVE);
+              case OBSERVABLE_GAUGE:
+                return AggregatorFactory.lastValue();
+            }
+            logger.log(Level.WARNING, "Unable to find default aggregation for instrument: " + i);
+            return null;
+          });
+  private static final Aggregation SUM = sum(AggregationTemporality.CUMULATIVE);
+  private static final Aggregation LAST_VALUE =
+      Aggregation.make("lastValue", unused -> AggregatorFactory.lastValue());
+  private static final Aggregation EXPLICIT_BUCKET_HISTOGRAM =
+      explictBucketHistogram(
+          AggregationTemporality.CUMULATIVE, DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES);
+
+  private Aggregation() {}
+
+  /**
+   * Returns the appropriate aggregator factory for a given instrument.
+   *
+   * @return The AggregatorFactory or {@code null} if none.
+   */
+  public abstract AggregatorFactory config(InstrumentDescriptor instrument);
+
+  /** The None Aggregation will ignore/drop all Instrument Measurements. */
+  public static Aggregation none() {
+    return NONE;
+  }
+
+  /** The default aggregation for an instrument will be chosen. */
+  public static Aggregation defaulAggregation() {
+    return DEFAULT;
+  }
+
+  /** Instrument measurements will be combined into a metric Sum. */
+  public static Aggregation sum(AggregationTemporality temporality) {
+    return Aggregation.make("sum", unused -> AggregatorFactory.sum(temporality));
+  }
+
+  /** Instrument meaasurements will be combined into a metric Sum. */
+  public static Aggregation sum() {
+    return SUM;
+  }
+
+  /** Remembers the last seen measurement and reports as a Gauge. */
+  public static Aggregation lastValue() {
+    return LAST_VALUE;
+  }
+
+  /**
+   * Aggregates measurments into an explicit bucket histogram using the default bucket boundaries.
+   */
+  public static Aggregation explictBucketHistogram() {
+    return EXPLICIT_BUCKET_HISTOGRAM;
+  }
+
+  /**
+   * Aggregates measurments into an explicit bucket histogram using the default bucket boundaries.
+   *
+   * @param temporality Whether to report DELTA or CUMULATIVE metrics.
+   */
+  public static Aggregation explictBucketHistogram(AggregationTemporality temporality) {
+    return explictBucketHistogram(temporality, DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES);
+  }
+
+  /**
+   * Aggregates measurments into an explicit bucket histogram.
+   *
+   * @param temporality Whether to report DELTA or CUMULATIVE metrics.
+   * @param bucketBoundaries A list of (inlcusive) upper bounds for the histogram. Should be in
```

Comment:
```suggestion
   * @param bucketBoundaries A list of (inclusive) upper bounds for the histogram. Should be in
```

@jsuereth Your spell-checker keeps on turning itself off? :P

Reply:
looks like it is broken after the latest update, let me figure out how I messed that up.